cmd/ore/aws/upload.go

```diff
@@ -76,6 +77,7 @@ func init() {
 	cmdUpload.Flags().StringVar(&uploadAMIDescription, "ami-description", "", "description of the AMI to create (default: empty)")
 	cmdUpload.Flags().StringSliceVar(&uploadGrantUsers, "grant-user", []string{}, "grant launch permission to this AWS user ID")
 	cmdUpload.Flags().BoolVar(&uploadCreatePV, "create-pv", false, "create a PV AMI in addition to the HVM AMI")
+	cmdUpload.Flags().StringSliceVar(&uploadTags, "tags", []string{}, "list to tags to attach to the AMI")
```

Comment:
Should document that this takes `key=value` pairs.

Reply:
Documented this in the example, but IMO we should have a detailed documentation for the commands. I could not find one, blame my poor if it already exists.

Reply:
The help text is the primary detailed documentation right now.